faust/assignor/client_assignment.py

```diff
@@ -143,10 +144,11 @@ def _colocated_partitions(
         return next(valid_partitions, set())
 
 
-class ClientMetadata(Record,
-                     serializer='json',
-                     include_metadata=False,
-                     namespace='@ClientMetadata'):
+class ClientMetadata(
+    Record,
+    include_metadata=False,
+    namespace='@ClientMetadata'
+):
```

Comment:
While I'm personally a huge fan of Black-style code, Faust has its own code style and I'd expect to get merged this code should follow that style.